app/templates/base.html

```diff
@@ -62,6 +64,16 @@ <h2 class="footer--heading">Related links</h2>
 							<a class="footer--link" href="https://www.civilservice.lgbt/mentoring" title="Find out more about our mentoring programme">
 								Our mentoring programme
 							</a>
+            </li>
+						<li class="footer--list-item">
+							<a class="footer--link" href="/privacy-and-data" title="Read the cookies policy">
```

Comment:
Should this say 'Read the privacy and data policy' ?

Reply:
Yes. Will fix.

Reply:
Fixed in fddf1b8